components/namespace-lister/base/deployment.yaml

```diff
@@ -40,10 +40,10 @@ spec:
         resources:
           limits:
             cpu: 500m
-            memory: 128Mi
+            memory: 256Mi
```

Comment:
As long as we're increasing things, maybe let's set the memory limit to something much higher than what we're requesting?

```suggestion
            memory: 1Gi
```

Reply:
makes sense. commited